command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 10:29:56AM -0700, Mike Brown wrote:
> We already have a `runtime` spec.  This is the `runtime CLI spec`.

If you want to draw those boxes, I think we want separate repositories
[1].  I'm also fine keeping them both in runtime-spec and having that
be a single spec [1].  I think keeping both in runtime-spec and *only*
drawing a box around the command-line spec is not internally
consistent [1].

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72818875